linked_list/implementation.py

```diff
@@ -1,40 +1,180 @@
-from .interface import AbstractLinkedList
-
+from linked_list.interface import AbstractLinkedList
+from linked_list.node import Node
 
 class LinkedList(AbstractLinkedList):
-    """
-    Implementation of an AbstractLinkedList inteface.
-    """
-
     def __init__(self, elements=None):
-        pass
+        self.start = None
+        self.end = None
+        
+        if elements:
+            for element in elements:
+                self.append(element)
 
+            
     def __str__(self):
-        pass
+        current_node = self.start
+        return_list = []
+        while current_node is not None:
+            return_list.append(current_node.elem)
+            current_node = current_node.next
+        
+        return str(return_list)    
```

Comment:
Great idea of making an actual list out of the LL node elements and then getting the string representation of that! You can make the code even more concise by using a list comprehension:
`return str([node.elem for node in self])